schema.graphql

```diff
@@ -15,8 +15,9 @@ type Account @entity {
 }
 
 type RoleAssignment @entity {
-  id: ID! # grantorAddress + tokenId + tokenAddress + granteeAddress + roleHash
+  id: ID! # rolesRegistryAddress + grantorAddress + tokenId + tokenAddress + granteeAddress + roleHash
   role: Role!
+  rolesRegistry: RolesRegistry!
```

Comment:
Since `RoleAssignment` already points to `Role` which points to `RolesRegistry`, is it really necessary for `RoleAssignment` to also point to `RolesRegistry`?